projects/src/transmitter.cc

```diff
@@ -0,0 +1,157 @@
+#include "transmitter.h"
+#include "hal.h"
+#include "packet/serialize.h"
+#include "usbconfig.h"
+
+// This define can be useful when sizing message mailbox and memory pools
+#define ASSERT_MESSAGE_MEMORY_LIMIT FALSE
```

Comment:
If you're running with debug asserts enabled, you can allow it to drop packets or not drop packets. Release builds should always drop packets.